pom.xml

```diff
@@ -1034,6 +1031,13 @@ limitations under the License.
                             </execution>
                         </executions>
                     </plugin>
+		    <plugin>
+		      <groupId>org.apache.maven.plugins</groupId>
+		      <artifactId>maven-javadoc-plugin</artifactId>
+		      <configuration combine.self="override">
+			<doclint>none</doclint>
+		      </configuration>
+		    </plugin>
```

Comment:
I went with the "override plugin config in a profile" approach used here because I couldn't find anywhere in the current Maven build that explicitly declares or configures the maven-javadoc-plugin.  I figured this approach would allow us to "fix" any Javadoc build no matter how it's invoked.